### PR TITLE
Introduce presto reader for paimon

### PR DESCRIPTION
**Pull request by s7monk**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,623 @@
...
+                                </resourceBundles>
+                            </configuration>
+                        </execution>
+                    </executions>
+                </plugin>
+            </plugins>
+        </pluginManagement>
+    </build>
+
+    <repositories>
```
Comment (JingsongLi) [inline pom.xml:611]: remove this?
see: https://github.com/apache/incubator-paimon/pull/927

**Review by JingsongLi — COMMENTED** [review]
We need a `README.md`.

File: pom.xml (hunk 1 above)
Comment (s7monk) [reply pom.xml:611]: ok

**Comment by s7monk** [comment]
> We need a `README.md`.

ok

**Review by JingsongLi — COMMENTED** [review]
You can refer to https://github.com/apache/incubator-paimon/commit/49491377b93223e914f5656d8d2c2f7ade7999bb to add snapshot publish.

**Review by JingsongLi — APPROVED** [review]
Looks good to me!

**Pull request merged**